soc/intel/intel_socfpga_std/cyclonev/Kconfig.soc

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Description:
+# Adding support to Cyclone V SoC FPGA
+
+config SOC_SERIES_CYCLONE5
+	bool
+	help
+	  "Intel SoC FPGA Cyclone5 Series"
+
+config SOC_SERIES
+	default "cyclonev" if SOC_SERIES_CYCLONE5
+
+config SOC_CYCLONE5
+	bool
+	help
+	  "Intel SoC FPGA Cyclone5"
+
+config SOC
+	default "intel_socfpga_std_cyclonev" if SOC_CYCLONE5
```

Comment:
```suggestion
	default "cyclone5" if SOC_CYCLONE5
```

Reply:
why to rename as `cyclone5` ?
what is wrong with `intel_socfpga_std_cyclonev`, or would it be `cyclonev` ?

Reply:
`	default "cyclonev" if SOC_CYCLONEV` can do if 5 is updated to V

Reply:
@nordicjm,  now I'm lost (( can you please point me where it is set as mandatory that all these names must match exactly to each other for v.2 ?

Reply:
Correct me if I'm wrong but @nashif was complaining about the length of board names and `cyclonev_socdk/cyclonev` seems a lot easier to type than `cyclonev_socdk/intel_socfpga_std_cyclonev`

Reply:
> Correct me if I'm wrong but @nashif was complaining about the length of board names and `cyclonev_socdk/cyclonev` seems a lot easier to type than `cyclonev_socdk/intel_socfpga_std_cyclonev`

This board has no other SoC or variants and its SoC name seems to follow naming convention to include vendor and SoC family/series.
Currently this PR generates almost the same .config file except `BOARD_CVSXDEVKIT` which looks like a leftover and conflicts with the v.2 scheme board name. 

>  .. can do if 5 is updated to V

From what I see from the SoC description `V` is a part of its official name, although `cyclone5` naming is also used in .dts, 
m.b. @mevalver can explain why there is this difference.

In my opinion adhoc renaming or optimizing should be minimal for this HWMv2 migration effort unless some vendor decides to apply its thoroughly developed naming scheme to align all its SoC/boards in the process.

Reply:
@golowanow no difference, its just a naming thing, you can called cyclone5 or cyclonev

Reply:
> @golowanow no difference, its just a naming thing, you can called cyclone5 or cyclonev

@mevalver, there is similar and related renaming question in regard of `./arm64/intel_socfpga_agilex_socdk` boards
https://github.com/zephyrproject-rtos/zephyr/pull/68909#issuecomment-1948592044

In case of `intel_socfpga_std` there are at least 2 options:

1) just keep naming 'as-is' and what  this PR does at the moment:
SoC - `intel_socfpga_std_cyclonev`, board - `cyclonev_socdk`
but using HWMv2 full `board/soc` naming it looks redundant: `cyclonev_socdk/intel_socfpga_std_cyclonev`

2) rename the SoC/board to some short form and in alignment with what might be chosen for `intel_socfpga`:
`intel_socfpga_std/cyclonev_socdk`
or:
`intel_socfpga_std_socdk/cyclonev`
or:
`intel_socfpga_std/cyclonev`

Reply:
name and text should match so e.g. `default "cyclone5" if SOC_CYCLONE5` or `default "cyclonev" if SOC_CYCLONEV`